pkg/codegen/python/gen.go

```diff
@@ -50,6 +51,12 @@ type typeDetails struct {
 
 type imports codegen.StringSet
 
+// defaultMinPythonVersion is what we use as the minimum version field in generated
+// package metadata if the schema does not provide a vaule. This version corresponds
+// to the minimum supported version as listed in the reference documentation:
+// https://www.pulumi.com/docs/reference/pkg/python/pulumi/
+const defaultMinPythonVersion = ">=3.6"
```

Comment:
Shouldn't this be 3.7? We don't support 3.6

Reply:
I pulled v3.6 from the docs website XD